packages/server/tests/acceptance/hbarLimiter.spec.ts

```diff
@@ -220,36 +220,49 @@ describe('@hbarlimiter HBAR Limiter Acceptance Tests', function () {
         global.accounts.push(...accounts);
       });
 
-      beforeEach(async function () {
-        const basicPlans = await hbarSpendingPlanRepository.findAllActiveBySubscriptionTier(
-          [SubscriptionTier.BASIC],
-          requestDetails,
-        );
-        for (const plan of basicPlans) {
-          await hbarSpendingPlanRepository.delete(plan.id, requestDetails);
-          await ethAddressSpendingPlanRepository.deleteAllByPlanId(plan.id, 'before', requestDetails);
-          await ipSpendingPlanRepository.deleteAllByPlanId(plan.id, 'before', requestDetails);
-        }
+      afterEach(async () => {
+        hbarSpendingPlanRepository.resetAmountSpentOfAllPlans(requestDetails);
+
+        // Note: Since the total HBAR budget is shared across the entire Relay instance by multiple test cases,
+        //       and expense updates occur asynchronously, the wait below ensures that the HBAR amount has sufficient time
+        //       to update properly after each test.
+        await Utils.wait(1500);
```

Comment:
nit: as usual I'm not a fan of waits.
Hmm, this means that we're going to add 1.5 * number of tests to this execution run time right?
I guess if it's only for this class it's acceptable right now to unblock but we should come back at some point to make this more stable without a wait

Reply:
Understood! I agree we should comeback to make it stable without waits. Will create a ticket for it.

Reply:
Created a ticket for it https://github.com/hashgraph/hedera-json-rpc-relay/issues/3199